app/src/main/res/values-ru/strings.xml

```diff
@@ -131,5 +131,6 @@
   <string name="operations">Операции</string>
 
   <string name="start_earlier_end">Дата конца не может быть раньше даты начала</string>
+    <string name="notes">примечания</string>
```

Comment:
Nit: The spacing is slightly off, and the first letter should be capitalized.

Reply:
Fixed with [abfe703](https://github.com/xorum-io/open_money_tracker/pull/34/commits/abfe7038194b61e69fa5eb42c158a3bee0ca592a)